### ISSUE TITLE
BSD-4-Clause text is damaged

### ISSUE THREAD

**Issue by pombredanne** [spdx/license-list-XML#1296]
"This product includes software developed by the the organization." is not right

**Comment by jlovejoy** [comment]
can you provide a bit more detail as to what you think is not right and why?

**Comment by pombredanne** [comment]
"This product includes software developed by `the the` organization." is not right because there is a repetition of the word `the`. I found that while running a routine scancode-toollkit scan on the license used by google-license-classifier to ensure that they would be mapped alright for https://github.com/nexB/scancode.io/pull/226

Since they use mostly SPDX I was expecting to have an exact match to the reference license texts in scancode.

And weirdly this should have fixed in https://github.com/spdx/license-list-XML/commit/9ae4d59f4f677000e49aeade9672fbfe003d3a8f 

I guess this typo in SPDX made it into the wild before that as it can be found in several license detection tools
https://github.com/search?q="by+the+the+organization"&type=code

The likely reason is that it is still present in these SPDX places: 
- https://github.com/search?p=1&q=org%3Aspdx+"the+the+organization"&type=Code


- https://github.com/spdx/license-list-data/blob/28c40746cd140f2d54b3ef62143df0f650751f82/rdfnt/BSD-4-Clause.nt
- https://github.com/spdx/license-list-data/blob/28c40746cd140f2d54b3ef62143df0f650751f82/json/details/BSD-4-Clause.json
- https://github.com/spdx/license-list-data/blob/28c40746cd140f2d54b3ef62143df0f650751f82/website/BSD-4-Clause.json
- https://github.com/spdx/license-list-data/blob/28c40746cd140f2d54b3ef62143df0f650751f82/jsonld/BSD-4-Clause.jsonld
- https://github.com/spdx/license-list-data/blob/28c40746cd140f2d54b3ef62143df0f650751f82/rdfturtle/BSD-4-Clause.turtle
- https://github.com/spdx/license-list-data/blob/28c40746cd140f2d54b3ef62143df0f650751f82/rdfxml/BSD-4-Clause.rdf
- https://github.com/spdx/license-list-data/blob/28c40746cd140f2d54b3ef62143df0f650751f82/text/BSD-4-Clause.txt

It likely was introduced  in https://github.com/spdx/license-list-XML/commit/8b640ed4ba3723111d276c573ff74a813a5e2764 when converting to XML from the original template syntax in https://github.com/spdx/license-list/blob/19b646901f00083e3100c110fa0211bbd043dfda/BSD-4-Clause.txt#L10

**Comment by goneall** [comment]
I looked at the source for [BSD-4-Clause.xml](https://github.com/spdx/license-list-XML/blob/master/src/BSD-4-Clause.xml) and it looked correct.

I finally figured out the problem.  The test text file, which is supposed to contain the original text for the license, still includes the repeated words.

LicenseListPublisher [PR#83](https://github.com/spdx/LicenseListPublisher/pull/83) fixed a long running complaint on text formatting by copying the text from the test files.  This change introduced the double word back into generated license texts.

I'll create a PR to fix.
